packages/runtime/src/widgets/Card.tsx

```diff
@@ -41,13 +44,17 @@ export const Card: React.FC<CardProps> = ({ children, styles }) => {
   const renderedChildren = useMemo(() => {
     return EnsembleRuntime.render(children);
   }, [children]);
-  const mergedStyles = merge(defaultStyles, styles);
+
+  const mergedStyles = { ...defaultStyles, ...styles };
   const { shadowOffset, shadowBlur, shadowSpread, shadowColor } = mergedStyles;
+
   return (
     <div
+      className={styles?.names}
       style={{
-        ...mergedStyles,
+        display: "grid",
```

Comment:
Why display: grid here?

Reply:
cause default block display isn't letting `gap` property have any effect

Reply:
We can use display: flex